src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,15 @@
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+        if (email == null) {
+            return null;
```

Comment:
Do we need this check? What will be the result of the expression `user.getEmail().equals(email)` if email == null?